tests/pytest_tests/unit_tests/test_data_types.py

```diff
@@ -1458,6 +1458,7 @@ def test_partitioned_table():
 ################################################################################
 
 
+@pytest.mark.wandb_core_failure()
```

Comment:
it used to work

Reply:
yeah i'll remove this, I realized after that it's actually just order-dependent